src/bundle/Resources/config/services/tabs/content_type.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    EzSystems\EzPlatformAdminUi\Tab\ContentType\:
+        resource: "../../../lib/Tab/ContentType/*"
+        parent: EzSystems\EzPlatformAdminUi\Tab\AbstractTab
```

Comment:
Please use `AbstractEventDispatchingTab` as a parent instead. It has been recently merged #728